README.md

```diff
@@ -14,16 +14,16 @@ It is the default face of the Material Design Specification, and available to ev
 
 In 2016 the OpenType format was [updated](https://medium.com/@tiro/https-medium-com-tiro-introducing-opentype-variable-fonts-12ba6cd2369) with a revised version of Apple's “TrueType GX” run-time interpolation technology.
 The primary motivation for the OpenType specification authors to do this was obtaining dramatic reductions in the file-size of large font families, such as Google's [Noto](https://github.com/googlei18n/noto-fonts/tree/master/phaseIII_only/unhinted/variable-ttf) fonts. 
-The Google Fonts team intends to go beyond compression: To make Variable Fonts meaningful.
+The Google Fonts team intends to go beyond compression: They want to make variable fonts *meaningful*.
 
-Variations technology offers the potential of new ways to do typography, and could improve Material Specification Typography guidance in myriad ways.
+Variable fonts create a new dimension in typographic possibilities, and could improve Material Specification Typography guidance in myriad ways.
```

Comment:
I agree that "Variable fonts" is smoother than "Variations technology", but I think 'potential' is the key idea.